akka-http-core/src/main/resources/reference.conf

```diff
@@ -233,6 +233,15 @@ akka.http {
       tcp-no-delay = undefined
     }
 
+    # Client https proxy options. If a setting is left undefined/-1, an error will be thrown
+    # indicating that the options must be set.
```

Comment:
Seems a bit weird - I mean, only if I actually want to use the proxy these should be required, reword the requirement somehow?